sudospawner/mediator.py

```diff
@@ -50,12 +49,19 @@ def kill(pid, signal):
     app_log.debug("Sending signal %i to %i", signal, pid)
     try:
         os.kill(pid, signal)
-    except OSError as e:
-        if e.errno == errno.ESRCH:
-            # not running
-            alive = False
-        else:
-            raise
+    except ProcessLookupError:
+        # not running
+        alive = False
+    except PermissionError:
```

Comment:
I don't know if `os.kill` can raise any other exceptions. Should the code here have a final fallback `except Exception` case which logs the unexpected exception and moves on as if the process is dead?